### ISSUE TITLE
Which version is recommended to develop against? latest or next?

### ISSUE THREAD

**Issue by XuluWarrior** [tauri-apps/tauri#3296]
Due to the audit freeze, `latest` is now 5 months old.  Since then a lot of fixes have been made in `next`.

We are new to tauri and hoped that our webapp would just drop in and work but quickly came across numerous issues with window events not firing as expected.

A number of these have been fixed in `next` but not all.

What is the status of `next`?  Once the audit is complete, will it be merged into `latest`?

Should people hoping to get to know tauri and building a production app once it's ready be working with `latest` or `next`?  Or would they be better off holding off all together until the audit is done and 1.0 is released?

**Comment by XuluWarrior** [comment]
For the record and those who bump into window event issues, these are the ones that I've found in latest but are fixed in next.
- These events don't fire in `1.0.0-beta.8` (In Rust or JS)
  - 'tauri://close-requested',
  - 'tauri://focus',
  - 'tauri://blur'
- In`1.0.0-beta.8` (in Rust), `app.listen_global` callbacks don't get called

**Comment by FabianLars** [comment]
the plan is to have everything in `next` part of 1.0. Same for the audit changes (not public yet).
We're getting pretty close to having the first release candidate ready, so if you want to start _right now_, imo (if possible) it's better to start directly with the `next` branch for an easier switch to 1.0-rc, especially if your app doesn't even work correctly with beta.8.

**Comment by XuluWarrior** [comment]
That's useful to know.
Thanks for the quick update.
